src/Symfony/Component/Cache/Traits/FilesystemTrait.php

```diff
@@ -36,7 +36,7 @@ public function prune()
                 continue;
             }
 
-            if ($time >= (int) $expiresAt = fgets($h)) {
+            if (0 < ($expiresAt = fgets($h)) && $time >= $expiresAt) {
```

Comment:
`if (($expiresAt = (int) fgets($h)) && $time >= $expiresAt) {`
the `isset($expiresAt[0])` below looks unneeded now, it's always true now
(same comments on the second change)

Reply:
Done …